api/src/main/java/com/codeforcommunity/api/ISiteProcessor.java

```diff
@@ -1,20 +1,20 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
-import com.codeforcommunity.dto.site.FavoriteSitesResponse;
+import com.codeforcommunity.dto.site.AdoptedSitesResponse;
 import com.codeforcommunity.dto.site.RecordStewardshipRequest;
 import com.codeforcommunity.dto.site.StewardshipActivitiesResponse;
 
 public interface ISiteProcessor {
 
   /** Creates a record in the favorite sites table linking the user and the site */
```

Comment:
Should rename to adopted sites table in comments

Reply:
Good catch, fixed